Library/Homebrew/test/cmd/list_spec.rb

```diff
@@ -18,4 +34,25 @@
       .and not_to_output.to_stderr
       .and be_a_success
   end
+
+  it "lists the formulae installed on request or automatically",
+     :integration_test do
```

Comment:
Don't think it's worth adding another (slow, sadly) integration test for this.

Reply:
Tangential: Now that each command is a separate class, do you think it'd be possible to create integration-like tests that don't require spinning up a new Ruby process? It might have been possible before but now it seems like a more viable option. Most of the slowness has to do with shelling out to a new Ruby process, right?

CC: @dduugg

Reply:
@apainintheneck Yeh, I think this would be nice.

We still want to make sure we have some proper/slow integration coverage because it catches when the issues are in `bin/brew` or `bin/brew.rb` but that could be a smaller subset of critical commands e.g. `brew install`/`brew upgrade`/etc.